test/cli/files.js

```diff
@@ -127,6 +127,23 @@ describe('files', () => runOnAndOff((thing) => {
       })
   })
 
+  it('add with piped argument', () => {
+    // echo 'src/init-files/init-docs/readme' | jsipfs files add
+    return ipfs('files add', { piped: 'src/init-files/init-docs/readme' })
```

Comment:
I'm unable to actually test this due to conflicts, if those could be resolved, would be 👍 

Also, judging by this test, the output of `echo 'src/init-files/init-docs/readme' | jsipfs files add` should place `src/init-files/init-docs/readme` as the content, without any filename, but the assertion seems to say that it just added a file called `readme` which is incorrect.